testingHelpers.js

```diff
@@ -25,7 +25,19 @@ function randomInnosentStream(min, max, length) {
   return array
 }
 
+async function clientFactory(db, overrides) {
+  const hasOverrides = typeof overrides === 'object' && overrides !== null && !Array.isArray(overrides)
+
+  // prettier-ignore
+  const client = await db.createClient(
+    hasOverrides && overrides.displayName || 'factoryClient',
+    hasOverrides && overrides.fromPhoneNumber || '+15558881234',
+  )
+  return client
+}
+
```

Comment:
What do you think of this structure? The idea is that it's a single function call that will populate the row with decent default values, but will allow you to overwrite any of them as you need to for the test that uses it.

If you like it, I think it would be very useful for creating Locations.

Reply:
So overrides is an object like 
{displayName: foo, fromPhoneNumber: bar} where we specify  the values we care about?

I think it could make the tests read much nicer! Could eliminate lots of boiler plate while focusing attention on the fields we care about.